File: 2.0/java/campaigns/postCallCampaignSound.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.CallCreateSound;
import com.callfire.api.client.api.common.model.ResourceId;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        CallCreateSound callCreateSound = new CallCreateSound();
        callCreateSound.setName("Sound 1");
        callCreateSound.setToNumber("12135551122");
        ResourceId resourceId = callfireClient.campaignSoundsApi().recordViaPhone(callCreateSound);
    }
}

File: 2.0/csharp/campaigns/postCallCampaignSound.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model;
using CallfireApiClient.Api.Common.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var callCreateSound = new CallCreateSound
        {
            Name = "Sound 1",
            ToNumber = "12135551122"
        };
        ResourceId resourceId = client.CampaignSoundsApi.RecordViaPhone(callCreateSound);
    }
}
